testing/e2e/agent_install_test.go

```diff
@@ -304,13 +270,13 @@ func (suite *AgentInstallSuite) TearDownTest() {
 	if suite.T().Skipped() {
 		return
 	}
-	path, err := exec.LookPath(agentName)
-	if err != nil {
-		suite.T().Logf("unable to detect elastic-agent install on test tear-down: %s", err)
-		return
-	}
-	err = exec.Command("sudo", path, "uninstall", "--force").Run()
-	suite.Require().NoError(err, "elastic-agent uninstall failed.")
+	//path, err := exec.LookPath(agentName)
+	//if err != nil {
+	//	suite.T().Logf("unable to detect elastic-agent install on test tear-down: %s", err)
+	//	return
+	//}
+	out, err := exec.Command("sudo", "elastic-agent", "uninstall", "--force").CombinedOutput() // TODO windows uninstall?
+	suite.Require().NoErrorf(err, "elastic-agent uninstall failed. Output: %s", out)
```

Comment:
At this point the suite is already tearing down: having a require to stop the test is not very useful, maybe use a simple assert